app/views/admin/workshops/show.html.haml

```diff
@@ -42,10 +42,12 @@
         %h3
           %small #{humanize_date(@workshop.date_and_time, @workshop.ends_at, with_time: true)}
         - if @workshop.rsvp_opens_at
-          RSVPs will open at
-          = @workshop.rsvp_opens_at.strftime('%H:%M on %d/%m/%Y.')
-          %br
-          %br
+          %div{style: "margin-bottom: 2em"}
```

Comment:
This looks great but can the css be moved in a class in one of the sass files instead - if one does not exist with such a property. This is so we can both reuse it but also not litter the views with inline styling.

Reply:
Sure! I wasn't sure what to do since I thought foundation would have utility classes but I couldn't find them. Should CSS always be one-offs or do we want utility classes such as `margin-bottom-2`?

Reply:
I think a utility class would be better in this instance.

Reply:
```suggestion
          %div.mb2
```